core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/colsAtAnyDepth.kt

```diff
@@ -69,11 +75,12 @@ public interface ColsAtAnyDepthColumnsSelectionDsl {
     /**
      * ## Cols At Any Depth
      *
-     * Returns the columns of this [ColumnsResolver] at any depth if they satisfy the optional given predicate.
+     * Returns all columns in [this\] at any depth (so also inside [Column Groups][ColumnGroup]) if they satisfy the
+     * optional given predicate.
      *
-     * This function is especially powerful if followed by another filter function such as
-     * [colsOf][ColumnsSelectionDsl.colsOf], [single][ColumnsSelectionDsl.single], and [valueCols][ColumnsSelectionDsl.valueCols].
-     * ### Check out: [Usage]
+     * This function is especially powerful if followed by another [ColumnSet] filter function like
```

Comment:
` This function is especially powerful if followed by another ` - it sounds a little bit emotional for technical documentation